Header header

float64[6] zeta
float64[3] tau_c
float64[3] nuC
